src/main/java/org/jabref/gui/LibraryTab.java

```diff
@@ -809,6 +809,11 @@ private void onClosed(Event event) {
         if (tableModel != null) {
             tableModel.unbind();
         }
+
+        if (stateManager.getOpenDatabases().isEmpty() && stateManager.activeDatabaseProperty().isPresent().get()) {
```

Comment:
This check is new.

I miss a "why". I think, it is because of the welcome screen. Then, please, add it as comment.

    // This check is needed, because the WelcomeTab is a tab, but not a database.